backend/controllers/authController.js

```diff
@@ -51,9 +56,21 @@ function logIn(req, res) {
     });
 }
 
-const authController = {
-    register: register,
-    logIn: logIn
+//Get authenticated user
+function getAuthUser(req, res) {
+    User.findOne({
+        where: {
+          email: req.user.email
+        }
+    }).then(user => {
+          console.log("user: ", user);
```

Comment:
[Suggestion] Maybe we could remove unnecessary console.log's for cleaner code

Reply:
I agree with that ! Did not see this unnecessary console.log, thanks for the suggestion !